jdwp/common.py

```diff
@@ -0,0 +1,61 @@
+"""Basic types for JDWP messages."""
+
+from typing import List
+
+
+class Types:
```

Comment:
Might be a good idea to turn this into an enum, perhaps `PrimitiveType`. Later we will need to include some more complex types, such as arrays and tagged unions and it might make sense to define `Type` as a union type:
```
Type = PrimitiveType | Array | TaggedUnion
```